Loop/Views/GlucoseHUDView.swift

```diff
@@ -65,19 +64,19 @@ final class GlucoseHUDView: HUDView {
         }
     }
 
-    func set(_ glucoseValue: GlucoseValue, for unit: HKUnit, from sensor: SensorDisplayable?) {
+    func set(glucoseQuantity: Double, at glucoseStartDate: Date, for unitDisplayString: String, from sensor: SensorDisplayable?) {
```

Comment:
Style nit from Swift's API design guidelines: `for unitDisplayString` should be just `unitDisplayString`, because the `String` type doesn't hold semantic meaning.